apple/RNCWebViewImpl.m

```diff
@@ -170,10 +170,10 @@ - (instancetype)initWithFrame:(CGRect)frame
     _autoManageStatusBarEnabled = YES;
     _contentInset = UIEdgeInsetsZero;
     _savedKeyboardDisplayRequiresUserAction = YES;
-#if !TARGET_OS_OSX
+#if TARGET_OS_IOS
     _savedStatusBarStyle = RCTSharedApplication().statusBarStyle;
     _savedStatusBarHidden = RCTSharedApplication().statusBarHidden;
-#endif // !TARGET_OS_OSX
+#endif // !TARGET_OS_IOS
```

Comment:
Does this need to remove the `!` to match?

Reply:
Good call, I missed it